dim/src/errors.rs

```diff
@@ -47,6 +47,8 @@ pub enum DimError {
     InvalidCredentials,
     #[error(display = "Requested username is not available.")]
     UsernameNotAvailable,
+    #[error(display = "Paths are already added to a library")]
```

Comment:
I'd change this to something else. Maybe "Paths have already been scanned" or "Paths exist in the database".

Reply:
Frontend actually uses the "Library" word, so that's why I've picked this error message, to help the user, where he should look. But I'm not a native speaker, so it was just a try :)